forms-flow-theme/scss/_modal.scss

```diff
@@ -593,3 +593,202 @@
     border: none;
   }     
 }
+
+
+//expression-builder-modal
+
+  .expression-list-group {
+    width: 100%;
+  }
+
+  .expression-list-group .form-variable {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    padding: 0.625rem;
+  }
+
+  .expression-list-group .form-variable:hover {
+    background: var(--ff-base-100);
+   
+    .variable-label {
+      color: var(--ff-primary) !important;
+      
+    }
+    .variable-key{
+      color: var(--ff-gray-400) !important;
+      
+    }
+  }
+  .expression-list-group .form-variable.selected-variable {
+    background: var(--ff-base-600);
+  }
+
+  .expression-list-group .variable-label {
+    color: var(--ff-primary);
+    font-size: var(--ff--font-size-sm);
+    font-weight: var(--font-weight-lg);
+  }
+
+  .expression-list-group .variable-key {
+    color: var(--ff-gray-400);
+    font-size: var(--ff--font-size-sm);
+    font-weight: var(--font-weight-sm);
+    margin-left: 8px;
+  }
```

Comment:
--ff--font-size-sm variable dont exist . please use --font-size-sm in both cases

Reply:
Resolved